mercury-wallet/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: mercury-wallet_web_1
+      APP_PORT: 3000
+      
+  web:
+    image: layer2tech/mercury-wallet-ui:latest

```

Comment:
We can't accept `latest` as a tag for Docker images, otherwise updates will get automatically rolled out whenever a user restarts their Umbrel server. Instead, we typically see developers using the app version as the tag. e.g. `layer2tech/mercury-wallet-ui:v0.8.6`.

We'll also require a multi-arch Docker image digest here too. You can get this by running `docker pull {image}`